neptune-bulk-loader/task.ts

```diff
@@ -54,10 +55,19 @@ export const neptuneBulkLoader = async (): Promise<MainResult> => {
 
   await createTermsIndex(esUrl, termsIndex);
 
+  const ontologyGraph = `<${metadata.ontologyGraphUrl}>`;
+
   await osClient.deleteByQuery(esUrl, termsIndex, {
-    match: {
-      graphUri: metadata.namedGraphUri,
-    },
+    match: { ontologyGraph },
+  });
+
+  await indexTerms({
+    elasticsearchUrl: esUrl,
+    ontologyNameSpace: metadata.ontologyNameSpace,
+    ontologyGraph,
+    terminologyTitle: 'my-terminology-title',
```

Comment:
Should this come from the metadata?

Reply:
It's not part of the metadata schema right now but how else would we get it? Either it has to come from the actual ontology (we'd need a query for that) or we just ask them to provide it as part of the metadata (like the namespace).

Reply:
This was fixed in e13e1f5